### DISCUSSION TITLE
241118_월

### DISCUSSION THREAD

**Discussion by minrongtic** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#20] [TIL]
**TO DO** 

<br> 
<details>
  <summary> <b>Segmentation 강의</b> </summary>
  
- [x] 3강 
- [ ] 7강 
- [ ] 8-1강 
- [ ] 8-2강

</details>

<details>
  <summary> <b>✅개발자스럽게git사용하기 강의</b> </summary>
  
- [x] 커밋 메세지 
- [x] 풀리퀘
- [x] github action
- [x] readme 

</details>

- [x] ai stages 홍석님 리팩토링 코드 + Recsys 리팩토링 코드 비교 분석 
- [ ] medSAM2 baseline 분석 
- [ ] wandb integration 해보기 + 종민쓰 wandb 사용법 따라해보기

**Comment by sejongmin** [comment]
- [x] 강의 마무리하기
- [x] 과제 마무리하기
- [x] 퀴즈 마무리하기
- [x] 13주차 개인 회고 마무리하기
- [x] wandb 정리하기
- [x] horizontal flip offline augmentation

**Comment by Yoon0717** [comment]
- [x] 베이스코드에 Transform 추가 및 수정 -> albumentation외에 사용할게 없어서 일단은 가볍게 구현
- [ ] Validation 최적화 해보기 (aistages 게시판 방법은 문제가 발생)
- [x] GPU 최적화 해보기 -> empty cache 위치 수정으로 해결

**Reply by minrongtic** [reply]
GPU 최적화 신기해용 어떻게 하는 건가용

**Reply by Yoon0717** [reply]
저도 잘 몰라서 torch.cuda.empty_cache 사용해서 메모리 캐시 해제하거나, 안쓰는 텐서 삭제하는거 해보면서 찾고 있어요

**Comment by Soy17** [comment]
- [ ] 강의 듣기 마무리
- [x] 프로젝트 UNet관련 튜닝 방향
- [x] 다들 아프지 말기 ~~

**Comment by wonjeongjeong** [comment]
- [x] 7,8 강의 듣기
- [ ] 퀴즈 풀기
- [ ] 강의에 나온 논문 구현 시도
